Source/Editor/Foundation/ResourceBrowserTab.cpp

```diff
@@ -589,8 +589,10 @@ void ResourceBrowserTab::RenderDirectoryContentEntry(const FileSystemEntry& entr
     if (!isCompositeFile)
         flags |= ImGuiTreeNodeFlags_Leaf;
 
-    const ea::string name = Format("{} {}", GetEntryIcon(entry, isCompositeFile), entry.localName_);
-    const bool isOpen = ui::TreeNodeEx(name.c_str(), flags);
+    constexpr size_t nameBufferSize = FILENAME_MAX + 5; // Icon and space take 4 extra characters + one character for zero at the end of the string.
```

Comment:
Just use string concatenation

Reply:
`ea::string` also has `sprintf`-like functions, could use those as well. I would also avoid having statics and add `ea::string tempBuffer_;` to the class itself and use that.